SimG4CMS/Calo/src/HGCalNumberingScheme.cc

```diff
@@ -51,7 +51,7 @@ uint32_t HGCalNumberingScheme::getUnitID(int layer, int module, int cell, int iz
       cellV = HGCalTypes::getUnpackedCellV(cell);
     } else if (mode_ != HGCalGeometryMode::Hexagon8) {
       double xx = (pos.z() > 0) ? pos.x() : -pos.x();
-      hgcons_.waferFromPosition(xx, pos.y(), layer, waferU, waferV, cellU, cellV, waferType, wt, false, false);
+      hgcons_.waferFromPosition(xx, pos.y(), layer, waferU, waferV, cellU, cellV, waferType, wt, false, true);
```

Comment:
No effect of changing this boolean? I don't see what about it in the PR description.

Reply:
This was a debug switch. I was debugging partial wafers. Now I have put it back to false